src/components/project-details/project-details.tsx

```diff
@@ -3,21 +3,40 @@
  * See the LICENSE file in the repository root folder for details.
  */
 
-import { Project } from 'utils';
+import { Project } from 'utils/src/types/project-types';
```

Comment:
```suggestion
import { Project } from 'utils';
```
Make sure to always be importing from the package and not the file itself.